src/backend/_scss/application.scss

```diff
@@ -129,7 +129,7 @@ a {
 
 /* Navbar adjustments */
 .logo {
-  max-height: 45px;
```

Comment:
Yes, we do need to set max-width and max-height.
So, if you set an inline HTML `style` `<img src="" width="{{width from API}}"/>` then HTML automatically maintains an aspect ratio of 1:1 to set the height, and we don't need to worry about this.
You can't hardcode width and height in CSS